src/main/resources/tags/info/version.tag

```diff
@@ -0,0 +1,10 @@
+tpye: text
+aliases: ver
+
+---
+
+Geyser is built to only support the latest major release of Bedrock at any given time. Betas are not supported, and older version are not supported for maybe more than a minor revision or two for a limited time.
```

Comment:
```suggestion
Geyser is built to only support the latest stable release of Bedrock at any given time. Betas are not supported, and older versions are not supported for maybe more than a minor revision or two for a limited time.
```

Reply:
I used "Major" because that means the first two number sets in the version string (i.e. 1.17) whereas anything past that is a _minor_ version.

"Versions" should be updated however.